### ISSUE TITLE
[Automation - BUG] Audit Logs is not populating in Automation Center model driven app in CoE Main environment

### ISSUE THREAD

**Issue by bilodeauju** [microsoft/powercat-automation-kit#379]
### Describe the issue

Our organization has been testing the Automation kit for a few months and we noticed the Audit Logs section within the Automation Center app in the CoE Main environment always remains empty while everything else appears to be working as it should.

![image](https://github.com/microsoft/powercat-automation-kit/assets/81716086/58e9643c-c70e-467d-bbfd-ca6fd0f11f7d)

The Audit log table within the CoE Main solution is also empty. 
![image](https://github.com/microsoft/powercat-automation-kit/assets/81716086/99e7a1be-e122-4637-b753-647847eac7a2)

I verified auditing is enabled for the CoE Main and all Satellite environments. 
I reviewed all the setup documentation on https://learn.microsoft.com/en-us/power-automate/guidance/automation-kit/setup/setup-checklist and following pages but there appears to be nothing regarding how to setup this Audit log specifically.

Please advise what is required to get this Audit log data populated in the Automation Center app?

### Expected Behavior

Audit logs should appear in the Automation Center model driven app

### What component are you experiencing the issue with?

Automation Center

### What solution version are you using?

1.0.20230907.1 (September 2023)

### What app or flow are you having the issue with?

Automation Center model driven app

**Comment by praveen-pulagam-gl** [comment]
@bilodeauju In order to get the audit logs data, auditing should be configured for the environments. Can you confirm whether audit is enabled for the environments. Also, make sure users are assigned to the security role Desktop Flows Support Analyst.
	
Please find the below URL for more details on desktop flow audit logs
[https://learn.microsoft.com/en-us/power-automate/guidance/automation-kit/control-center-desktop-flow-audit-logs](https://learn.microsoft.com/en-us/power-automate/guidance/automation-kit/control-center-desktop-flow-audit-logs)

**Comment by bilodeauju** [comment]
Yes, as stated I already verified auditing was enabled for the CoE Main and for all the Satellite environments.
I will check regarding the security role Desktop Flows Support Analyst, thank you.

**Comment by bilodeauju** [comment]
@praveen-pulagam-gl The link you shared, https://learn.microsoft.com/en-us/power-automate/guidance/automation-kit/control-center-desktop-flow-audit-logs, this is for the **Control center** app. Please note that we have no problem with auditing in Control Center, it appears correctly there.

To clarify, we are only having problem with audit logs not appearing in **Automation Center app** in the CoE Main environment as per my screenshot.
